test/utils/image/manifest.go

```diff
@@ -69,7 +69,7 @@ func (i *Config) SetVersion(version string) {
 func initReg() RegistryList {
 	registry := RegistryList{
 		GcAuthenticatedRegistry: "gcr.io/authenticated-image-pulling",
-		E2eRegistry:             "gcr.io/kubernetes-e2e-test-images",
+		E2eRegistry:             "k8s.gcr.io/e2e-test-images",
```

Comment:
this change shouldn't be made. This would break any one that's running the GPU tests, that are using the ``cuda-vector-add:1.0`` image.

Reply:
Just an FYI, I'm taking care of this here: https://github.com/kubernetes/kubernetes/pull/100886

It's the 3rd PR in a series that takes care of the cuda-vector-add image.